### ISSUE TITLE
W10 / 17763 BSOD on Mono feature.

### ISSUE THREAD

**Issue by skiddyfree** [cheat-engine/cheat-engine#1428]
Title says all.

to reproduce
1. load dbk / load dbvm
2. opens unity-based game, try any mono-related feature
3. ???
4. BSOD!

**Comment by skiddyfree** [comment]
also, loading DBK / DBVM after using mono feature doesnt BSOD and mono feature itself does work properly.

**Comment by skiddyfree** [comment]
yeah well, mono related features are buggy as hell. especially it often throws LUA exceptions for null reference issue.

**Comment by skiddyfree** [comment]
yeah. can confirm you cant call mono functions / IL Disassembly :/

**Comment by cheat-engine** [comment]
how about just dbk?

**Comment by skiddyfree** [comment]
> 
> 
> how about just dbk?

in that case, using mono features doesnt crash but calling mono functions / IL Disassemble still doesnt works with LUA exception.

**Comment by cheat-engine** [comment]
is it il2cpp ? as that doesn't support it (no vtable)

**Comment by skiddyfree** [comment]
> 
> 
> is it il2cpp ? as that doesn't support it (no vtable and no il code)

nope, clean C# managed code.

i just noticed that it only happens if you turns dbk / dbvm first then mono feature. if you turnd them later, it doesnt BSODs even if you disables/enables mono feature again.

**Comment by skiddyfree** [comment]
let me know if you confirmed this bug, i'll close this.

**Comment by cheat-engine** [comment]
Not yet, but whenbi'm finished with the next function i'll give it a look.

which bsod do you get? clock timeout ?

**Comment by skiddyfree** [comment]
alright. 

no, it was nonpaged area bsod. so prob some code in driver is making this bsod.
